tools/codegen/core/experiments_compiler.py

```diff
@@ -217,8 +217,9 @@ def IsValid(self, check_expiry=False):
             print("expiry should be no more than two quarters from now")
         return not self._error
 
-    def AddRolloutSpecification(self, allowed_defaults,
-                                allowed_platforms, rollout_attributes):
+    def AddRolloutSpecification(
```

Comment:
Why are these file changes showing up in this PR ? They look like sanity check fixes

Reply:
weird, just synced/merged, looks back to normal